mne/stats/regression.py

```diff
@@ -392,3 +392,254 @@ def _make_evokeds(coefs, conds, cond_length, tmin_s, tmax_s, info):
             kind='average')  # nave and kind are technically incorrect
         cumul += tmax_ - tmin_
     return evokeds
+
+
+class EncodingModel(object):
+    def __init__(self, delays=None, est=None, preproc_y=True):
+        """Fit a STRF model.
```

Comment:
Oops - that's actually going to be changed to `EncodingModel`. I'm porting some of this over from STRF modeling code I wrote